src/__tests__/test-utils.test.tsx

```diff
@@ -0,0 +1,123 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { ComponentType } from "react";
+import { render } from "@testing-library/react";
+import { paramCase, pascalCase } from "change-case";
+import { describe, expect, test } from "vitest";
+
+import * as components from "../../lib/components";
+import createWrapperDom, { ElementWrapper as DomElementWrapper } from "../../lib/components/test-utils/dom";
+import createWrapperSelectors from "../../lib/components/test-utils/selectors";
+import { defaultProps } from "./default-props";
+
+const RENDER_COMPONENTS_DEFAULT_PROPS: Record<string, unknown>[] = [
+  {
+    "data-testid": "first-item",
+    "data-name": "first item",
+  },
+  {
+    "data-testid": "second-item",
+    "data-name": "second item",
+  },
+];
+
+function renderComponents(componentName: string, props = RENDER_COMPONENTS_DEFAULT_PROPS) {
```

Comment:
Will we extend it to more than CodeView? What to do if different components need different props?

Reply:
We have the same pattern in all of the repos (except `components`), which we read the needed props from `defaultProps` file (Line `25`).